cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/RunsCount/index.tsx

```diff
@@ -33,11 +33,7 @@ function getRunsCount(pipeline) {
   const program = getProgram(pipeline);
   const totalRuns = objectQuery(program, 'totalRuns');
 
-  if (totalRuns === null || totalRuns === undefined) {
-    return 0;
-  }
-
-  return totalRuns;
+  return totalRuns || 0;
```

Comment:
Was this change intended to be in this pull request?

Reply:
Yes, it came from rebasing with the branch of https://github.com/cdapio/cdap/pull/11486